package.json

```diff
@@ -118,7 +121,7 @@
     "concurrently": "8.2.2",
     "copyfiles": "2.4.1",
     "doctoc": "2.2.0",
-    "esbuild": "0.20.2",
+    "esbuild": "0.18.20",
```

Comment:
I have to downgrade it because `modern-node-polyfills` not support esbuild@0.18. But PR with fix is already merged. And I pinged the guys with a request to release new version